cornice_swagger/swagger.py

```diff
@@ -531,14 +532,20 @@ def _extract_path_from_service(self, service):
             Path definition.
         """
 
-        path = {}
+        path_obj = {}
+        path = service.path
+
+        # handle traverse and subpath as regular parameters
+        # docs.pylonsproject.org/projects/pyramid/en/latest/narr/hybrid.html
+        for subpath_marker in ('*subpath', '*traverse'):
+            path = path.replace(subpath_marker, '{subpath}')
```

Comment:
If we assume that `subpath` and `traverse` are always outside `{ .* }` we can do something like this.